_docs/ops/secrets.md

```diff
@@ -52,7 +76,8 @@ echo 'region = us-gov-west-1' >> ~/.aws/config
 echo "mfa_serial = $mfa_serial" >> ~/.aws/config
 ```
```

Comment:
@pburkholder I realize this isn't a part of your changes and something I had modified with the previous update, but I wonder if we should modify it still, especially after seeing Jonathan Pirro's demonstration using `aws-vault` for login.gov.  This will only configure a profile for the operator's production GovCloud account, correct?

Does it make sense to call that out and leave it as an exercise for the operator to determine for their remaining accounts, or expand on this a bit more with more detailed instructions?

Reply:
@pburkholder and I caught up about this, we'll address it separately from this work!